### DISCUSSION TITLE
Hello

### DISCUSSION THREAD

**Discussion by xxchan** [xxchan/xxchan#5] [General]
🫣

**Comment by rum1887** [comment]
Hi, I'm working out the details for a wasmedge project. 

Here's the link to the project: https://github.com/WasmEdge/WasmEdge/issues/2471

Can you explain what C based host APIs mean? Does it mean c based api to call the backend? Wasi-nn specification is implemented in Rust in wasmedge. I don't quite understand what the c based apis mean


Your insights will mean to lot to me🙂. Please help.
